elements/pfe-tabs/src/pfe-tab-panel.html

```diff
@@ -1,5 +1,5 @@
 <div tabindex="-1" role="tabpanel">
```

Comment:
Same suggestion here, no need to use the helper class approach in the templates.